src/bench/base58.cpp

```diff
@@ -13,7 +13,7 @@
 
 static void Base58Encode(benchmark::State& state)
 {
-    unsigned char buff[32] = {
```

Comment:
Is that because b+32 is passed to the EncodeBase58 function and clang thinks the function might use it?